tutorminio/patches/openedx-auth

```diff
@@ -1,3 +1,3 @@
 AWS_STORAGE_BUCKET_NAME: "{{ MINIO_BUCKET_NAME }}"
 AWS_S3_CUSTOM_DOMAIN: ""
-AWS_QUERYSTRING_AUTH: false
\ No newline at end of file
+AWS_QUERYSTRING_AUTH: {{ MINIO_QUERYSTRING_AUTH }}
```

Comment:
This will render to "False", and not "false". (note the upper case "F") "False" will be incorrectly evaluated to "true" in Python code. 

I think you need:

    {% if MINIO_QUERYSTRING_AUTH %}true{% else %}false {% endif %}

But then I'm wondering what's the purpose of the `MINIO_QUERYSTRING_AUTH`. What role is it playing for you?

Reply:
My comment above is still valid. The `MINIO_QUERYSTRING_AUTH` variable will not render to "false" or "true".